packages/tiler-sharp/src/index.ts

```diff
@@ -63,16 +71,28 @@ export class TileMakerSharp implements TileMaker {
     return this.toImage(format, this.createImage(background).composite(layers));
   }
 
+  /** Can we just serve the image directly from the cotar */
+  private isDirectCotar(ctx: TileMakerContext): boolean {
+    if (ctx.layers.length !== 1) return false;
+    const firstLayer = ctx.layers[0];
+    if (firstLayer.type !== 'cotar') return false;
+    if (this.tileSize !== 256) return false; // TODO this should not be hard coded
+    if (ctx.format !== ImageFormat.Webp) return false; // TODO this should not be hard coded
```

Comment:
Are we able to convert the webp to png or jpeg before serving?

Reply:
this is only checking if we can directly serve the image back to the user without actually modifying it.

so using this path the server never even has to uncompress the webp!